Updates to VERBOSE_COMMANDS deploy and build variable description (#7711)
magento/devdocs: src/cloud/env/variables-build.md

-  **Default**—_Not set_
-  **Version**—Magento 2.1.4 and later

> Enable or disable the [Symfony](https://symfony.com/doc/current/console/verbosity.html) debug verbosity level for `bin/magento` CLI commands executed from the deployment script.

Reviewer: Using the word executed is not advised.
```suggestion
Enable or disable the [Symfony](https://symfony.com/doc/current/console/verbosity.html) debug verbosity level for `bin/magento` CLI commands performed during the deployment phase.
```

> The [MIN_LOGGING_LEVEL]({{ site.baseurl }}/cloud/env/variables-global.html#min_logging_level) must be set to `debug` for the most useful output.

Author: ```suggestion
{:.bs-callout}To use VERBOSE_COMMANDS to control the detail in command output for both failed and successful `bin/magento` CLI commands, you must set [MIN_LOGGING_LEVEL]({{ site.baseurl }}/cloud/env/variables-global.html#min_logging_level) `debug`.
```

Reviewer: That sounds much better.

> To change the verbosity of the output from _successful_ `bin/magento` CLI commands, you must also set the [MIN_LOGGING_LEVEL]({{ site.baseurl }}/cloud/env/variables-global.html#min_logging_level) to `debug`.

Reviewer: I don't think that verbosity is only for successful commands. I think this may be confusing. Perhaps keep it simple.

```suggestion
The [MIN_LOGGING_LEVEL]({{ site.baseurl }}/cloud/env/variables-global.html#min_logging_level) must be set to `debug` for the most useful output.
```

Author: #7708 was update from Cloud Devs that controlling the verbosity level of command output for successful commands requires the `debug` setting for MIN_LOGGING_LEVEL.

However, I think the simplification you suggested is fine.

Author: > I don't think that verbosity is only for successful commands. I think this may be confusing. Perhaps keep it simple.

According to update submitted by Cloud Devs in #7708, the detail level for the command output for successful commands requires setting min_logging_level to `debug`.    Which is confusing.  I simplified it as you suggested.
